giant/plugins/client/java_android/java_android/java_android/templates/sdk/src/main/java/com/yourcompany/packagename/operation/AthenaApiInterceptor.java.jinja

```diff
@@ -0,0 +1,124 @@
+{% import 'variables.jinja' as vars with context %}
+{% import 'variables-android.jinja' as android with context %}
+
+{{ (android.main_path + '/operation/AthenaApiInterceptor')|start_of_file -}}
+package {{ android.package_root_name }}.operation;
+
+import android.content.Context;
+import android.text.TextUtils;
+import android.util.Log;
+
+import com.lixar.athena.cache.CacheFactory;
+import com.lixar.athena.cache.RestCache;
+import com.lixar.athena.model.Entity;
+import com.lixar.athena.model.Resource;
+import com.lixar.athena.utility.LogUtil;
+import com.lixar.athena.utility.MoshiDateTimeAdapter;
+import com.squareup.moshi.JsonAdapter;
+import com.squareup.moshi.Moshi;
+
+import java.io.IOException;
+import java.util.Collections;
+import java.util.HashMap;
+import java.util.Map;
+
+import okhttp3.Headers;
+import okhttp3.Interceptor;
+import okhttp3.Request;
+import okhttp3.Response;
+import okhttp3.ResponseBody;
+
+public class AthenaApiInterceptor implements Interceptor {
```

Comment:
Please rename this. Either use `{{ vars.api_name }}ApiInterceptor`, or something generic like `GiantApiInterceptor`.